crates/cairo-lint-core/tests/test_files/loop_for_while/loop_for_while

```diff
@@ -34,6 +34,42 @@ fn main() {
 
 //! > ==========================================================================
 
+//! > loop with arithmetic condition and else block
+
+//! > cairo_code
+fn main() {
+    let mut x: u16 = 5;
+    loop {
+        if x * 2 >= 20 {
+            break;
+        } else {
+            x += 1;
+        }
+    }
+}
+
+//! > diagnostics
+warning: Plugin diagnostic: you seem to be trying to use `loop`. Consider replacing this `loop` with a `while` loop for clarity and conciseness
+  --> lib.cairo:4:5
+   |
+ 4 |       loop {
+   |  _____-
+ 5 | |         if x * 2 >= 20 {
+...  |
+ 9 | |         }
+10 | |     }
+   | |_____-
+   |
+
+//! > fixed
+fn main() {
+    let mut x: u16 = 5;
+    while x * 2 < 20 {
+    }
```

Comment:
this should contain what's in the `else` block
add also a test like that:
```cairo
fn main() {
    let mut x: u16 = 5;
    loop {
        if x * 2 >= 20 {
            break;
        } else {
            x += 1;
        }
        x += 1;
    }
}
```
and the fix should be something like that:
```cairo
fn main() {
    let mut x: u16 = 5;
    while x * 2 < 20 {
         x+=1;
         x+=1;
    }
}